jeffhuth-bytecode/tap-mambu
# v.1.0.2 Revert, re-apply pagination changes

# Description of change
Fix merge issue with `v.1.0.1`. Revert to `v.1.0.0 `and re-apply pagination changes.

# Manual QA steps
Tested singer-check-tap, discovery, initial load and incremental to target-stitch
 
# Risks
Low - reverting to fix prior merge issue.
 
# Rollback steps
Revert to v.1.0.0
